src/components/Header.tsx

```diff
@@ -1,3 +1,30 @@
-export const Header = () => {
-	return <h1>TO-DO-LIST</h1>;
+import "./header.css";
+import { Login } from "./Login";
+import { SignIn } from "./SignIn";
+
+interface propsH {
```

Comment:
nechal bych tady Props jenom, jelikoz to nikam neexportujes. Pokud bys to exportovala, tak bych dal HeaderProps. Zaroven pis typy Pascal casem, tj. velky pocatecni pismeno